gen/bonk.hoon

```diff
@@ -0,0 +1,7 @@
+::  Helm: bonk ames
+::
+::::  /hoon/bonk/hood/gen
+  ::
+/?    310
+:-  %say
+|=({^ ~ ~} helm-bonk+~)
```

Comment:
This generator should be in `gen/hood/`

Reply:
woops, copied that wrong out of the pier.